creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -105,7 +123,16 @@ const Metrics = Object.freeze({
             `STATE_MACHINE_${jobName}_JOB_DURATION_SECONDS_HISTOGRAM`
           ],
           help: `Duration in seconds for a ${jobName} job to complete`,
-          labelNames: ['uncaughtError'], // Whether the job completed (including with a caught error) or quit unexpectedly
+          labelNames: [
```

Comment:
hahaha i can see you had fun with all this templating

its pretty hard to follow whats happening and i think the problem is that - if someone were to look at this file, the expectation would be that all info about a metric should be located in one object, whereas currently they must scroll up and down the file to find what they need. i shouldn't have separated the `MetricLabels` and `MetricLabelNames`, it just adds complexity

if you wanna keep it, we can for now since its using an existing pattern and i know you enjoyed doing this hahah. can circle back later

Reply:
haha it was kinda fun but yeah I agree it's a little harder to parse through than we'd like. circling back later sounds good 👍